extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -11,63 +11,86 @@ import { context } from './testContext';
+		let expectedActions;
+		if (process.platform === 'win32') {
+			expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler', 'Properties'];
+		}
+		else {
+			expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler'];
+		}
+		const expectedString = expectedActions.join(',');
+		const actualString = actions.join(',');
+		assert(expectedActions.length === actions.length && expectedString === actualString, `Expected actions: "${expectedString}", Actual actions: "${actualString}"`);
+	}
 
-			let node = nodes[index];
-			let actions = await azdata.objectexplorer.getNodeActions(node.connectionId, node.nodePath);
-			let expectedActions;
+	@stressify({ dop: ObjectExplorerTester.ParallelCount, iterations: ObjectExplorerTester.IterationCount })
+	async azureDbNodeLabelTest() {
+		const expectedNodeLabel = ['Databases', 'Security'];
+		let server = await getAzureServer();
+		await this.VerifyOeNode(server, 3000, expectedNodeLabel);
+	}
 
-			if (process.platform === 'win32') {
-				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler', 'Properties'];
-			} else {
-				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler'];
-			}
+	@stressify({ dop: ObjectExplorerTester.ParallelCount, iterations: ObjectExplorerTester.IterationCount })
+	async standaloneNodeLabelTest() {
+		if (process.platform === 'win32') {
+			const expectedNodeLabel = ['Databases', 'Security', 'Server Objects'];
+			let server = await getStandaloneServer();
+			await this.VerifyOeNode(server, 3000, expectedNodeLabel);
+		}
+	}
 
-			const expectedString = expectedActions.join(',');
-			const actualString = actions.join(',');
-			assert(expectedActions.length === actions.length && expectedString === actualString, `Expected actions: "${expectedString}", Actual actions: "${actualString}"`);
-		});
-	});
-}
-async function VerifyOeNode(server: TestServerProfile, timeout: number, expectedNodeLabel: string[]) {
-	await connectToServer(server, timeout);
-	let nodes = <azdata.objectexplorer.ObjectExplorerNode[]>await azdata.objectexplorer.getActiveConnectionNodes();
-	assert(nodes.length > 0, `Expecting at least one active connection, actual: ${nodes.length}`);
+	@stressify({ dop: ObjectExplorerTester.ParallelCount, iterations: ObjectExplorerTester.IterationCount })
+	async bdcNodeLabelTest() {
+		const expectedNodeLabel = ['Databases', 'Security', 'Server Objects', 'Data Services'];
+		let server = await getBdcServer();
+		await this.VerifyOeNode(server, 6000, expectedNodeLabel);
+	}
+
+	async VerifyOeNode(server: TestServerProfile, timeout: number, expectedNodeLabel: string[]) {
```

Comment:
Function names should be camelCased

Reply:
Will change.

---
In reply to: [285805418](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r285805418) [](ancestors = 285805418)